frontend/src/app/components/wp-table/wp-table-configuration.ts

```diff
@@ -57,6 +57,12 @@ export class WorkPackageTableConfiguration {
   /** Whether this table is in an embedded context*/
   public isEmbedded:boolean = false;
 
+  /** Whether the number of shown WP per page shall be calculated based on the available height */
+  public calculatePerPage:boolean = false;
+
+  /** The calculated number of shown WP per page based on the available height */
+  public calculatedPerPageOption:number = 20;
```

Comment:
Can we turn this into a single attribute? I also prefer a different name other than `calculated` since it implies the table will do anything with that value.

`public forcePerPageOption:number|false = false`